src/domains/wallet/pages/WalletDetails/__snapshots__/WalletDetails.test.tsx.snap

```diff
@@ -1273,7 +1273,7 @@ exports[`WalletDetails should delete wallet 1`] = `
                       role="columnheader"
                     >
                       <div
-                        class="flex flex-inline align-top undefined"
+                        class="flex align-top flex-inline undefined"
```

Comment:
@brenopolanski in order this undefined to be avoided, I think if you provide an empty string when `column.ClassName` is empty here: https://github.com/ArkEcosystem/desktop-wallet/blob/4b6426b4bba1d133ef9858e652334f047725a8ec/src/app/components/Table/Table.tsx#L63
Would fix this issue

Reply:
Update: It is now patched in a separate PR: https://github.com/ArkEcosystem/desktop-wallet/pull/2540/

Reply:
Fixed here: https://github.com/ArkEcosystem/desktop-wallet/pull/2540